issue found on: temp/analyser.go:204:1: paramTypeCombine

an issue has been detected in source code: ./temp/analyser.go:204:1: paramTypeCombine: func(list1 []AggregatorLogEntry, list2 []AggregatorLogEntry) []AggregatorLogEntry could be replaced with func(list1, list2 []AggregatorLogEntry) []AggregatorLogEntry

--- a/temp/analyser.go
+++ b/temp/analyser.go
@@ -200,7 +200,7 @@ func messageWithOffsetIn(entries []AggregatorLogEntry, offset int) bool {
 	return false
 }
 
-func diffEntryListsByOffset(list1 []AggregatorLogEntry, list2 []AggregatorLogEntry) []AggregatorLogEntry {
+func diffEntryListsByOffset(list1, list2 []AggregatorLogEntry) []AggregatorLogEntry {
 	diff := []AggregatorLogEntry{}
 	for _, element := range list1 {
 		if !messageWithOffsetIn(list2, element.Offset) {

